### DISCUSSION TITLE
Why MaxCheckpointBatchSize in Azure Storage backend isn't exposed in host.json?

### DISCUSSION THREAD

**Discussion by jjshao-ms** [Azure/azure-functions-durable-extension#2442] [Q&A]
Can we set [MaxCheckpointBatchSize](https://github.com/Azure/durabletask/blob/53091265df19f18346b94465cc9ef95ffe8f4248/src/DurableTask.AzureStorage/AzureStorageOrchestrationServiceSettings.cs#L123) in host.json?
Our function has to use a lot of named pipe, causing the host to restart a lot. I think `MaxCheckpointBatchSize` could help a lot to avoid Activity rescheduled during restart. It would be great to be able to set `MaxCheckpointBatchSize`.

**Comment by cgillum** [comment]
@sebastianburckhardt is this something we can/should expose in host.json?

I see there is a `MaxEntityOperationBatchSize` setting that's visible to host.json, but not sure if it's the same thing or not.
